Dockerfile

```
@@ -1,4 +1,12 @@
 FROM google/dart:2.13
+
+WORKDIR /build
 COPY pubspec.yaml .
+COPY /lib ./lib/
+
 RUN dart pub get
+
+RUN cd lib && tar -czvf ../cdn_assets.tar.gz *.js
```

#supernit If we rearrange these commands, the `pub get` layer won't get invalidated by changes to `lib/`
```suggestion
RUN dart pub get

COPY /lib ./lib/
RUN cd lib && tar -czvf ../cdn_assets.tar.gz *.js
```